extensions/interactions/AlgebraicExpressionInput/directives/oppia-response-algebraic-expression-input.component.ts

```diff
@@ -1,4 +1,4 @@
-// Copyright 2020 The Oppia Authors. All Rights Reserved.
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
```

Comment:
I'm curious -- when exactly is this supposed to be done? When the file is created, when it gets updated, or is this change supposed to happen for all files every year?

Reply:
Good question. I am not so sure about this. Maybe @seanlip might know better. I usually update it when I am modifying a file.

Reply:
It doesn't need to be done.